### PR TITLE
Worker Info through Telegram.Bot

### PR DESCRIPTION
**Pull request by supertx2**
Uses the .NET Client for Telegram Bot API from https://github.com/TelegramBots/Telegram.Bot
to send the worker info when the user sends the command !info to the bot

### CODE REVIEW

**Comment by migrivas** [comment]
+1

**Comment by supertx2** [comment]
Added new Commands:
/running
/profit
/forcestop:(+workerName)
/forcestart:(+workerName)
/forcerestart:(+workerName)

(I'm Still testing this, change the time in  SendAutoInfo() - _AutoInfoTimerPeriod = 1000 * 20;)
/sendautoinfo
/stopautoinfo

**Comment by Ndekere** [comment]
Wow! Am FLOATING... But def sounds intriguing. I just started coding the other day. Hope to contribute to Github repos someday. 
Cheers,
amateur Programmer.

**Comment by emx0r** [comment]
hi. is there already compiled version I can try?

**Comment by supertx2** [comment]
Hi @emx0r , no, you need to compile it yourself, after that you can grab the .exe and the telegram .dll and paste it in the original compiled version

**Comment by emx0r** [comment]
@supertx2 thank you

**Comment by kramarenkop** [comment]
Привет @emx0r, нет, вам нужно скомпилировать его самостоятельно, после этого вы можете захватить .exe и телеграмма .dll и вставить его в исходную скомпилированную версию

File: NiceHashMiner/Configs/Data/GeneralConfig.cs [hunk 1]
```diff
@@ -61,6 +63,7 @@ public class GeneralConfig
         public double MinimumProfit = 0;
         public bool IdleWhenNoInternetAccess = true;
         public bool UseIFTTT = false;
+        public bool UseTelegramBot = false;
         public bool DownloadInit = false;
 
         public bool RunScriptOnCUDA_GPU_Lost = false;
```
Comment (kramarenkop) [inline NiceHashMiner/Configs/Data/GeneralConfig.cs:69]: >

**Review by kramarenkop — APPROVED** [review]

**Pull request closed without merging**